frontend/src/container/PlannedDowntime/PlannedDowntimeForm.tsx

```diff
@@ -348,6 +467,7 @@ export function PlannedDowntimeForm(
 						popupClassName="datePicker"
 					/>
 				</Form.Item>
+				<div className="scheduleTimeInfoText">{startTimeText()}</div>
```

Comment:
why this needs to be a function ? can we make a constant and memoize the same ?

Reply:
done